ts/Dashboards/Board.ts

```diff
@@ -831,6 +851,9 @@ namespace Board {
             },
             layouts: []
         },
+        editMode: {
+            enabled: true
```

Comment:
Change the API default (EditMode.ts file) and if possible move this piece of code there.

Reply:
I think that it's unnecessary, so we can remove this value and leave like it was.